File: app/src/main/java/com/sinhvien/webnovelapp/api/CoinPackage.java
package com.sinhvien.webnovelapp.api;

public class CoinPackage {
    private int id;
    private String name;
    private int coinAmount;
    private int bonusCoins;
    private double priceUSD;
    private double priceVND;
    private boolean isFeatured;

    public int getId() { return id; }
    public String getName() { return name; }
    public int getCoinAmount() { return coinAmount; }
    public int getBonusCoins() { return bonusCoins; }
    public double getPriceUSD() { return priceUSD; }
    public double getPriceVND() { return priceVND; }
    public boolean isFeatured() { return isFeatured; }
}

File: DarkNovel/DarkNovel/Models/CoinPackage.cs
namespace DarkNovel.Models
{
    public class CoinPackage
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public int CoinAmount { get; set; }
        public int BonusCoins { get; set; }
        public decimal PriceUSD { get; set; }
        public decimal PriceVND { get; set; }
        public bool IsActive { get; set; }
        public bool IsFeatured { get; set; }
        public int SortOrder { get; set; }
        public DateTime CreatedAt { get; set; }
        public DateTime? UpdatedAt { get; set; }
    }
}
